pkgs/by-name/on/onetbb/package.nix

```diff
@@ -105,5 +105,6 @@ stdenv.mkDerivation (finalAttrs: {
       thoughtpolice
       tmarkus
     ];
+    broken = hwloc.meta.broken;
```

Comment:
That statement seems unnecessary. What is the idea of putting it here?

Reply:
oh it is so `useTBB ? !onetbb.meta.broken` above works. I see now. The problem is if you use the `broken` field, that is just for the current package, and not the whole tree. It works for an immediate dependency but not transitive dependencies, so this line does the transitive part.

Reply:
> The problem is if you use the `broken` field, that is just for the current package, and not the whole tree.

Yeah, this is why it feels hacky to me.  It's not really scalable.